build.gradle.kts

```diff
@@ -56,7 +56,7 @@ subprojects {
 
         dependencies {
             // add("api", project(":tempest-bom"))
-            add("api", platform(Dependencies.nettyBom))
+            add("api", platform(libs.nettyBom))
```

Comment:
Hmm, I'm kind of surprised this works. In `subprojects` blocks you usually need to reference `libs` via `rootProject.libs` due to a gradle bug (https://github.com/gradle/gradle/issues/16634). Are you able to confirm this block is actually getting run?

Reply:
Hm, it must be - CI is passing and this would be a build-time error otherwise. Odd, not sure why it didn't work on other repos...

Reply:
Seems like it works inside the `plugins.withType` block but not outside. Woo gradle!